### PR TITLE
fix 404 in lambda authorizers blog

### PR DESCRIPTION
**Pull request by susanev**

### CODE REVIEW

**Review by kimberleyamackenzie — APPROVED** [review]

File: themes/default/content/blog/protecting-your-apis-with-lambda-authorizers-and-pulumi/index.md [hunk 1]
```diff
@@ -290,5 +290,4 @@ Get going with these resources:
 - [Getting Started]({{< relref "/docs/get-started" >}})
 - [Serverless REST API on AWS](https://github.com/pulumi/examples/tree/master/aws-ts-apigateway)
 
-To learn more about using Pulumi and Lambda authorizers
-[here](https://github.com/pulumi/pulumi-awsx/tree/master/nodejs/awsx/apigateway#lambda-authorizers).
+[Pulumi and Lambda authorizers](https://github.com/pulumi/pulumi-awsx/tree/master/awsx-classic/apigateway#lambda-authorizers).
```
Comment (cnunciato) [inline themes/default/content/blog/protecting-your-apis-with-lambda-authorizers-and-pulumi/index.md:293]: Do we need to remove the "learn more about" language?

```suggestion
[Learn more about Pulumi and Lambda authorizers](https://github.com/pulumi/pulumi-awsx/tree/master/awsx-classic/apigateway#lambda-authorizers).
```

**Review by cnunciato — CHANGES_REQUESTED** [review]
The link fix is correct, this is how things read with this change:

image

Can we find a way to keep the "learn more" language?

**Comment by susanev** [comment]
@cnunciato feel free to push a commit

**Review by cnunciato — APPROVED** [review]
Added a few words -- feel free to merge if you're okay with them!

**Pull request merged**